apps/daedalus_client/src/util.rs

```diff
@@ -7,7 +7,7 @@ use std::sync::Arc;
 use tokio::sync::Semaphore;
 
 lazy_static::lazy_static! {
```

Comment:
Since Rust 1.80 stabilized [`LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html#method.new), the `lazy_static` crate is no longer necessary, so we should remove our dependencies on it at some point.

That said, I don't think doing so is within the scope of this PR, so let's leave it as is for now. Just noting it here as a reminder for later.